OrbitCore/ElfFile.cpp

```diff
@@ -63,20 +99,21 @@ bool ElfFileImpl<ElfT>::IsAddressInTextSection(uint64_t address) const {
     return false;
   }
 
-  uint64_t section_begin_address = text_section_->getAddress();
-  uint64_t section_end_address =
-      section_begin_address + text_section_->getSize();
+  uint64_t section_begin_address = text_section_->sh_addr;
+  uint64_t section_end_address = section_begin_address + text_section_->sh_size;
 
   return (address >= section_begin_address) && (address < section_end_address);
 }
 
 template <typename ElfT>
 bool ElfFileImpl<ElfT>::GetFunctions(Pdb* pdb,
                                      std::vector<Function>* functions) const {
+  if (!contains_symbols_) {
+    return false;
+  }
```

Comment:
I am not sure about this - maybe we want to read .dynsym in this case - it does not provide full coverage of symbols but it does provide some. Which is better then none.

Add a todo maybe here - this can be done in a separate PR